recovery/redo/undo.go

```diff
@@ -0,0 +1,662 @@
+// Copyright 2019 The zbdba Authors
```

Comment:
The ungo.go file should only contain undo related content, but it also contains other types of redo records.